### PR TITLE
Use CSS in favor of <br> for display math

### PR DESCRIPTION
**Pull request by Aver1y**
Sorry for the delay

### CODE REVIEW

File: src/Text/Pandoc/Writers/HTML.hs [hunk 1]
```diff
@@ -1108,14 +1112,11 @@ inlineToHtml opts inline = do
...
-                            ! A.title (toValue str)
-              let brtag = if html5 then H5.br else H.br
-              return $ case t of
-                        InlineMath  -> m
-                        DisplayMath -> brtag >> m >> brtag
+              return $ imtag ! A.style "vertical-align:middle"
+                             ! A.src (toValue $ url <> T.pack (urlEncode (T.unpack $ s <> str)))
+                             ! A.alt (toValue str)
+                             ! A.title (toValue str)
+                             ! A.class_ mathClass
```
Comment (Aver1y) [inline src/Text/Pandoc/Writers/HTML.hs:1119]: This may be somewhat easy to miss in this diff, but the `A.class_ mathClass` here is new.

**Comment by jgm** [comment]
This looks good to me!  But it says there are conflicts with the base branch. Can you look into this?

**Comment by Aver1y** [comment]
Oh, yes some of the tests need fixing.

**Comment by jgm** [comment]
Thank you!

**Pull request merged**